### ISSUE TITLE
New technique: Cautious Weight Decay

### ISSUE THREAD

**Issue by Yura52** [kozistr/pytorch_optimizer#453]
A new technique ["Cautious Weight Decay"](https://arxiv.org/abs/2510.12402) has been recently proposed, which looks relevant to this repository.
